lightning/src/ln/offers_tests.rs

```diff
@@ -300,8 +300,12 @@ fn prefers_non_tor_nodes_in_blinded_paths() {
 	let tor = SocketAddress::OnionV2([255, 254, 253, 252, 251, 250, 249, 248, 247, 246, 38, 7]);
 	announce_node_address(charlie, &[alice, bob, david, &nodes[4], &nodes[5]], tor.clone());
 
+	let params = BlindedPathParams {
+		paths: PATHS_PLACEHOLDER,
```

Comment:
Here and throughout, we shouldn't change the number of paths used in the tests unless we are testing a specific scenario (which we should also do in a new test).

Reply:
Done! Update the file to use the default constructor instead of setting the number of paths using the global constant in [pr3246.06](https://github.com/shaavan/rust-lightning/commits/pr3246.06)
Thanks a lot for the suggestion!